app/views/layouts/_header.html.erb

```diff
@@ -0,0 +1,19 @@
+<div class="container col-lg-8 col-lg-offset-2">
+<header>
+    <div class="">
+        
+        <nav>
+            <%= link_to "Twitter Clone", root_path, id: "logo" %>
+            <ul id='nav'>
+                <li><%= link_to "Home", root_path %></li>
+                <% if !user_signed_in? %>
+                    <li><%= link_to "Log in", new_user_session_path %></li>
```

Comment:
Inconsistent quotation marks. Are we going with single quotes or double quotes? Pick one, and keep it consistent :)